lib/console_web/templates/email/api_key_email.html.eex

```diff
@@ -200,9 +200,9 @@
 
         <!-- Create white space after the desired preview text so email clients don’t pull other distracting text into the inbox preview. Extend as necessary. -->
         <!-- Preview Text Spacing Hack : BEGIN -->
-        <div style="display: none; font-size: 1px; line-height: 1px; max-height: 0px; max-width: 0px; opacity: 0; overflow: hidden; mso-hide: all; font-family: sans-serif;">
```

Comment:
this commenting is to fix the empty spaces in the subject for all files (this originally came from template)

Reply:
lmao wtf